mm2src/coins/eth/eth_tests.rs

```diff
@@ -213,30 +214,30 @@ fn test_wei_from_big_decimal() {
 
 #[test]
 #[ignore]
-/// temporary ignore, will refactor later to use dev chain and properly check transaction statuses
```

Comment:
Should we remove `#[ignore]` for this test now that the statuses are checked?

Reply:
I ignored these tests because they were successful individually but failing with cargo test, and I couldn't find out the reason. I used different addresses for them and now they're running successfully also with cargo test, so the problem was a race condition.